content/articles/compiled-redesign.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "CompilED for Inclusion"
+date: 2019-04-11
+type: "post"
+authors: ["mustapha"]
+tags: ["accessibility","inclusive design"]
+lede: "I restructured and redesigned the CTL DevTeam’s website CompilED to meet
+the web accessibility standards, and to improve its overall user experience
+inclusively on all devices."
+shortlede: "This site’s latest iteration has improved design and WCAG 2.0
+compliance."
+poster: "poster-compiled-v3.jpg"
+socmediaimg: ""
+hiliteimg: "poster-compiled-v3.jpg"
+poster_sourceurl: ""
+poster_source: ""
+bookendanimal: "dragon"
+---
+
+In March, I restructured and redesigned the CTL DevTeam’s website _CompilED_ to
+meet the web accessibility standards, and to improve its overall user
+experience inclusively on all devices.
```

Comment:
I'd remove 'inclusively' here, as 'all' is inclusive of every device.

Reply:
That word is purely jargon. I thought it would be nice to use it because it is in the title. But yes, inclusive == all-ish